apps/chat/src/utils/app/data/storages/api/conversation-api-storage.ts

```diff
@@ -113,3 +113,38 @@ export const getPreparedConversations = ({
         : path,
     };
   }); // to send conversation with proper parentPath and lastActivityDate order
+
+const isRootConversationsId = (id?: string) =>
+  isRootId(id) && id?.startsWith(`${ApiKeys.Conversations}/`);
+
+export const getImportPreparedConversations = ({
+  conversations,
+  conversationsFolders,
+}: {
+  conversations: Conversation[] | ConversationInfo[];
+  conversationsFolders: FolderInterface[];
+}) =>
+  conversations.map((conv) => {
+    const { path } = getPathToFolderById(
+      conversationsFolders,
+      conv.folderId,
+      true,
+    );
+
+    const newName = conv.name.replace(notAllowedSymbolsRegex, '');
```

Comment:
```suggestion
    const newName = prepareEntityName(conv.name);
```
name can't be more than 160 symbols now.

Reply:
Thank you for the suggestion!
New prepareEntityName function used in all needed places.